test/integration/authenticated-api/authenticated-api.test.ts

```diff
@@ -113,7 +113,7 @@ describe("AuthenticatedApi", () => {
       await axiosNoAuthInstance.get("route1");
       throw Error("Expected a 401 response");
     } catch (err) {
-      expect(err.message).toBe("Request failed with status code 401");
+      expect(err).toMatch("Request failed with status code 401");
```

Comment:
This works around the issue that `err` is `any`. A stringified Error will take format: `Error: {message}` so this will match.